### PR TITLE
Make ControlBarButton styling API consistent

### PR DESCRIPTION
**Pull request by prprabhu-ms**
# What

Small API consistency tweak.

# Why

**Is this a breaking change?**

- [x] This change causes current functionality to break.
 Renames a public type.

### CODE REVIEW

File: packages/react-components/src/components/ControlBarButton.tsx [hunk 1]
```diff
@@ -33,6 +34,13 @@ export interface ControlBarButtonStrings {
   offLabel?: string;
 }
 
+/**
+ * Styles for all {@link ControlBarButton} implementations.
+ *
+ * @public
+ */
+export interface ControlBarButtonStyles extends IButtonStyles {}
```
Comment (alcail) [inline packages/react-components/src/components/ControlBarButton.tsx:42]: is this for later on when (if) specific styles will be added?

Comment (alcail) [reply packages/react-components/src/components/ControlBarButton.tsx:42]: In the meantime, you could defined it like following to handle this issue:
export type ControlBarButtonStyles = IButtonStyles;

**Review by alcail — APPROVED** [review]

**Review by alkwa-msft — APPROVED** [review]

**Review by JamesBurnside — APPROVED** [review]
Not sure if linting/tsc will allow the empty interface in `export interface ControlBarButtonStyles extends IButtonStyles {}` but happy for this to be a type or interface and rest of changes all look great so approving

File: packages/react-components/src/components/ControlBarButton.tsx (hunk 1 above)
Comment (prprabhu-ms) [reply packages/react-components/src/components/ControlBarButton.tsx:42]: Lint allows empty interfaces, and this pattern seems to be common in calling SDK.

The intention, that we are extending the interface is made more explicit that way.

Also, changing `type` to `interface` later would be a breaking change, I think.

**Pull request merged**